hypervisor/include/common/ptdev.h

```diff
@@ -14,7 +14,7 @@
 
 #define INVALID_PTDEV_ENTRY_ID 0xffffU
 
-enum ptdev_vpin_source {
+enum ptirq_vpin_source {
```

Comment:
Suggest to add a note to describe what "ptirq_xxxx" stantds for.
No need to describe every structure, a note for the format of "ptirq_xxx" is enough.

Reply:
will submit another patch to update API/structure description in doxygen format.